### ISSUE TITLE
Philips Ambx Support

### ISSUE THREAD

**Issue by Wolfieeewolf** [zegreatclan/SimHub#293]
Any change of getting the Philips Ambx fans, light and rumble bar to work with SimHub?

There is an SDK. A bit hard to find but if you need a copy email me and I'll send it to you

**Comment by zegreatclan** [comment]
Hi ! I did not knew about this product, however I took a look and it's not sold anywhere, so I guess Philips discontinued it from what I see. I'm affraid I can't work on something I can't get my hand on it, I'm sorry.

**Comment by Wolfieeewolf** [comment]
Thanks for trying, I know the SDK has a virtual Ambx setup built in so you can test it but just not in a physical aspect. 

the hardware is difficult to get a hold of. You catch it on ebay now and again. Some are way over priced.  I have a few linked together but being so old not many games are supported anymore :(

If you lived in Australia i'd send you a set to test with but postage cost way too much for me to send this overseas.

I'd be happy to test if you wanted to have a look at the SDK but i understand if you need the hardware for testing purposes your end.

**Comment by Wolfieeewolf** [comment]
SDK if you want it

[amBX SDK (February 2009).zip](https://github.com/zegreatclan/SimHub/files/2593274/amBX.SDK.February.2009.zip)

**Comment by Wolfieeewolf** [comment]
amBX For VB.net

[ambx.zip](https://github.com/zegreatclan/SimHub/files/2593285/ambx.zip)

**Comment by zegreatclan** [comment]
Hi ! I took a second look, I've installed all the softwares required, it is possible indeed to do it , But the fact that that I can't get the hardware on my ends is really a deal breaker, I've tried a few times to do "blind" development in the past, but when it comes the time for support, it makes things hell. Simhub is evolving very often and I need to test everything again regularly with code changes. I'm sorry I really can't go on this way, even if this hardware looked pretty interesting.
